### PR TITLE
16 UI token page

### LINKED ISSUE #16: UI - token page
- [ ] vertically align (left-justified) token icons and names
- [ ] on token creation page, creation and token tables should be horizontally aligned

image

### PR DESCRIPTION
**Pull request by Viterbo**
# Description

Fixes #16 (which includes #51 #52 #54 #55)
Improvements to the look and feel and interaction of the listing, editing, and token creation pages.
The work focused on the re-implementation of many graphical and functional aspects of the token-details component.

## Changes
Changes are described in detail with screenshots in the corresponding issue
- Icons and name alignment - #51
- tables not aligned -  #52
- token-detail re-implementation - #54
- footer buttons on edit/create form #55

## Screenshots
Here's an overall screenshot but every detailed screenshot is on every issue
image

### CODE REVIEW

File: src/pages/tokens/components/token-edit.vue [hunk 1]
```diff
@@ -235,9 +237,52 @@
...
+
+@media (min-width: $breakpoint-md-min)
+  .form-container
+    min-width: 900px
+</style>
+
 <script>
 import TokenDetail from "./token-detail.vue";
 import { mapState, mapGetters, mapActions } from "vuex";
+const defaultIcon =
```
Comment (donnyquixotic) [inline src/pages/tokens/components/token-edit.vue:284]: can we move this into assets or public and import it accordingly?

**Review by donnyquixotic — CHANGES_REQUESTED** [review]

**Comment by Viterbo** [comment]
TODO: need to fix
- [ ] footer buttons need some top margin
- [ ] small logo not shown while token creation or edition
- [ ] **max supply** and **current supply** are not shown while token creation
- [ ] **default icon** in the token edit component should use an URL instead of inline image data encoding.

File: src/pages/tokens/components/token-edit.vue (hunk 1 above)
Comment (Viterbo) [reply src/pages/tokens/components/token-edit.vue:284]: got it!
I included it in the TODO list below.

**Comment by Viterbo** [comment]
This commit [818c6f8](https://github.com/telosnetwork/app-telos-native/pull/59/commits/818c6f82f8bba990567d332224fd8a1ebebe2821) would be the last one for this UI Token Page list of issues.

**Review by donnyquixotic — APPROVED** [review]

**Pull request merged**